feeds/feed.go

```diff
@@ -18,20 +15,21 @@ type Package struct {
 	Type        string    `json:"type"`
 }
 
-func NewPackage(created, cutoff time.Time, name, version, feed string) (*Package, error) {
-	if created.Before(cutoff) {
-		return nil, fmt.Errorf("package was created before cutoff time: %s", cutoff.String())
-	}
-	log.WithFields(log.Fields{
```

Comment:
Should this log be re-added to the ApplyCuttoff func?

Reply:
This is added to `scheduler.go` as part of `Poll()`

Reply:
I don't have a strong preference but it is a change in behaviour as it was only logged after the cutoff was applied

Reply:
Cutoff is applied inside the feeds. Only post-cutoff packages are logged. The difference in behaviour is the timing of the logging, logging occurs after all packages have been fetched for a given feed instead of during processing.

Reply:
Yeh fair enough, no user-facing difference here :+1: